tex/paper.tex

```diff
@@ -329,27 +330,173 @@ \subsection{Description of the Axelrod python package}\label{sec:description-of-
-* TFT, T2FT, TF2T -- no so great but variants with e.g. deadlock breaking, such as OmegaTFT,
-fare well
-* It's posible to win many matchups as well score highly -- backstabber, doublecrosser,
-foolmeonce. These strategies also score well in noisy tournaments
-* It's possible to be generally cooperative and still exploit naive strategies effective,
-such as LookerUp and MetaHunter
-* Long memory strategies are generally better
-* Meta strategies fare very well in general, also in noisy tournaments
-* Variance of score distributions is surprisingly low
-
-Plot of memory_depth vs mean score, mean wins?
+In parallel to the Python library, a tournament is being kept up to date that
+pits all strategies against each other. Figure~\ref{fig:tournament} shows the
+results from the full tournament.
+
+\begin{figure}[!hbtp]
+    \centering
+    \includegraphics[width=.8\textwidth]{../img/tournament.pdf}
+    \caption{Results from the library tournament (2015-12-19)}
+    \label{fig:tournament}
+\end{figure}
+
+The current winning strategy is a novel strategy of type: \textbf{LookerUp}. This
+is a strategy that maps a given set of states to actions. The state space is
+defined generically by \(m, n\) so as to map states to actions in the following
+way:
+
+\[(\underbrace{(C, D, D, D, C, D, D, C)}_{m\text{ first actions by opponent}},
+\overbrace{((C, C), (C, C))}^{n\text{ last pairs of actions}}) \to D\]
+
+The above example is an incomplete illustration of the mapping for \(m=8, n=2\).
+Intuitively this state space uses the initial plays of the opponent to gain some
+information about the intentions of it whilst still taking in to account the
+recent play.  The actual winning strategy an instance of the this framework for
+\(m=n=2\) for which an evolutionary algorithm has been used to train it. More
+details of this can be found in \cite{mojones}. In \cite{} experiments are
+carried out to evaluate how this particular strategy behaves in environments
+other than those in which it was trained\dots
+
+There are various other insights that have been gained from ongoing open
+research on the library. These include:
+
+\begin{itemize}
+    \item A closer look at zero determinant strategies, showing that
+        extortionate strategies obtain a large number of wins \textit{but} do
+        not perform particularly well. This is relevant given the findings of
+        \cite{Stewart2012} in which zero determinant strategies are shown to be
+        able to perform better than a given other strategy. This finding extends
+        to noisy tournaments (which are also implemented in the library).
+    \item This negative relationship between wins and performance does not
+        generalise, there are some strategies that perform well, both in terms
+        of matches won and overall performance: Back stabber, Double crosser and
+        Fool Me Once. These strategies continue to perform well in noisy
+        tournaments.
```

Comment:
However, some of these strategies have knowledge of the length of the game (BackStabber and DoubleCrosser), but this is not necessary to rank well in both wins and score as demonstrated by LookerUp and FoolMeOnce.